roles/node_images_hypervisor/files/management-vm/isolated.xml

```diff
@@ -0,0 +1,8 @@
+<network>
+  <name>isolated</name>
+  <ip address='192.168.254.1' netmask='255.255.255.252'>
+    <dhcp>
+      <range start='192.168.254.2' end='192.168.254.2' />
```

Comment:
For the management VM, we may want to specify the IP address to use? Are you accomplishing that by setting the range to one IP? I guess that works...

Reply:
That makes sense. I'll do that.